product/optee-stm32mp1/fw/config_scmi_clocks.c

```diff
@@ -0,0 +1,254 @@
+/*
```

Comment:
Please change SMCI to SCMI in the title  "product/optee-stm32mp1: expose SMCI clocks"

Reply:
ok